packages/custom-webpack/src/custom-webpack-builder.ts

```diff
@@ -10,15 +10,16 @@ export const defaultWebpackConfigPath = 'webpack.config.js';
 type CustomWebpackConfig =
   | Configuration
   | Promise<Configuration>
-  | ((baseWebpackConfig: Configuration, buildOptions: any) => Configuration)
-  | ((baseWebpackConfig: Configuration, buildOptions: any) => Promise<Configuration>);
+  | ((baseWebpackConfig: Configuration, buildOptions: any, targetOptions: any) => Configuration)
+  | ((baseWebpackConfig: Configuration, buildOptions: any, targetOptions: any) => Promise<Configuration>);
```

Comment:
Please add proper typing (`TargetOptions`, not any). It would be also nice to fix the typing for `buildOptions`.

Reply:
Fix and added to PR